src/java/org/sakaiproject/feedback/tool/entityproviders/FeedbackEntityProvider.java

```diff
@@ -136,6 +136,11 @@ private String handleReport(final EntityView view, final Map<String, Object> par
         final String screenSize = screenWidth + " x " + screenHeight + " pixels";
         final String plugins = (String) params.get("plugins");
         final String ip = requestGetter.getRequest().getRemoteAddr();
+        int fmt = DateFormat.MEDIUM;
+        Locale locale = sakaiProxy.getLocale();
```

Comment:
I've used DateForamt.MEDIUM here as it seemed the most appropriate looking at what the options look like